src/test/scala/cognite/spark/v1/FlexibleDataModelCorePropertyRelationTest.scala

```diff
@@ -51,13 +51,13 @@ class FlexibleDataModelCorePropertyRelationTest
   private val viewNodesListAndNonListExternalId = "sparkDsTestViewNodesListAndNonList2"
   private val viewEdgesListAndNonListExternalId = "sparkDsTestViewEdgesListAndNonList2"
 
-  private val viewAllAmbiguousTypeExternalId = "sparkDsTestViewAllAmbiguousType5"
-  private val viewNodesAmbiguousTypeExternalId = "sparkDsTestViewNodesAmbiguousType5"
-  private val viewEdgesAmbiguousTypeExternalId = "sparkDsTestViewEdgesAmbiguousType5"
+  private val viewAllAmbiguousTypeExternalId = "sparkDsTestViewAllAmbiguousType6"
+  private val viewNodesAmbiguousTypeExternalId = "sparkDsTestViewNodesAmbiguousType6"
+  private val viewEdgesAmbiguousTypeExternalId = "sparkDsTestViewEdgesAmbiguousType6"
 
-  private val viewAllTypeExternalId = "sparkDsTestViewAllType5"
-  private val viewNodesTypeExternalId = "sparkDsTestViewNodesType5"
-  private val viewEdgesTypeExternalId = "sparkDsTestViewEdgesType5"
+  private val viewAllTypeExternalId = "sparkDsTestViewAllType6"
+  private val viewNodesTypeExternalId = "sparkDsTestViewNodesType6"
+  private val viewEdgesTypeExternalId = "sparkDsTestViewEdgesType6"
 
```

Comment:
Might be just my new tests that are badly setup bc they are the main culprit for flakyness?